hints_tests.py

```diff
@@ -33,7 +33,7 @@ def test_program(program_name: str):
     test_program("memset")
     test_program("dict_new")
     test_program("dict_read")
-    # test_program("dict_write") # ValueError: Custom Hint Error: AttributeError: 'PyTypeId' object has no attribute 'segment_index'
+    test_program("dict_write") # ValueError: Custom Hint Error: AttributeError: 'PyTypeId' object has no attribute 'segment_index'
```

Comment:
Should we remove the comment with the error?

Reply:
Removed! https://github.com/lambdaclass/cairo-rs-py/pull/118/commits/0c6e4407564840b5d53be07e89161214729f6110